packages/sentry/sentry.js

```diff
@@ -1,11 +1,11 @@
 import * as Sentry from '@sentry/browser';
 
 export const boot = () => {
-  if (window.app.env === 'production') {
+  if (window.app.trackingSentry) {
```

Comment:
To keep it simple, we only need to check for the `trackingSentry` url value. For example, we might also include staging (previously it was only enabled for production).